kubernetes/romana.rc

```diff
@@ -3,13 +3,13 @@ select 0
 stuff "/bin/root -c /home/ubuntu/romana.conf^M"
 screen -t romana-topology
 select 1
-stuff "/bin/topology -rootUrl http://__MASTER_IP__:9600^M"
+stuff "/bin/topology -rootURL http://__MASTER_IP__:9600^M"
```

Comment:
`stuff` puts things into screen's input buffer (as if a user typed it). `^M` is the carriage return.